gfx/UltimateCataclysm/pngs_normal_32x32/overmap/public_garden/public_garden.json

```diff
@@ -0,0 +1,4 @@
+[{
+	"id": "publicgarden",
```

Comment:
sorry to bother you with unimportant things again, but is there a reason the png can't be named `publicgarden`?

Reply:
Why couldn't the ID be named public_garden? I guess it doesn't matter

Reply:
I meant that it would make this file unneeded. But ok, that's not an issue.

Reply:
Oh yeah sorry, for some reason I didn't realize that it could be done without json